http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```
@@ -566,6 +566,7 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
         dbRef.inTransaction {
           clusterQuery.mergeInstances(runtimeAndRuntimeConfig.runtime.copy(dataprocInstances = dataprocInstances))
         }.void
+      case CloudService.AzureVm => F.unit
```

throw exception here suggesting this should never happen instead of doing nothing silently